CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Expand print link component ([PR #2900](https://github.com/alphagov/govuk_publishing_components/pull/2900))
+* Add link click tracking ([PR #2904](https://github.com/alphagov/govuk_publishing_components/pull/2904))
```

Comment:
Nitpick - can this match the PR name? (either change it here or rename the PR is fine)